### DISCUSSION TITLE
Using gnuplot-mode on windows is problematic.

### DISCUSSION THREAD

**Discussion by bruceravel** [emacs-gnuplot/gnuplot#90] [Q&A]
A fellow names John Hardy (among several who have contacted me over the last couple of years) said:

You probably get nagged quite a bit about this. Some have been running into issues with gnuplot-mode and Windows and I’m wondering if you have any comments. I use it just fine on Linux, but it seems that Windows users are not able to send data to gnuplot successfully.

Org-mode also uses gnuplot-mode and this org-plot and org-mode babel+gnuplot are not working correctly on Windows.

Any thoughts? I’m hoping to include any results in the Org-mode wiki for gnuplot use: http://orgmode.org/worg/org-contrib/babel/languages/ob-doc-gnuplot.php

If you’re interested, here are some of the threads I’m referring to: 
- http://newsgroups.derkeiler.com/Archive/Comp/comp.emacs/2007-07/msg00159.html
- http://www.mail-archive.com/emacs-orgmode@gnu.org/msg14544.html
- http://groups.google.com/group/gnu.emacs.help/browse_thread/thread/53489131c79f62b3

If you’d like to see my summary of the issues to the org-mode mailing list, it’s here: http://thread.gmane.org/gmane.emacs.orgmode/30235

**Comment by BrianZbr** [comment]
I'm having trouble even figuring out how to install gnuplot-mode in Windows. Judging from this thread I guess it's not worth trying any further?

**Comment by joddie** [comment]
It's easy to install by enabling MELPA (see http://melpa.milkbox.net/), followed by `M-x list-packages` and choosing `gnuplot` from the list. Or you can just download all the `.el` files from the github repository, byte compile them and put somewhere in your Emacs load-path.

That said, there do seem to be problems on Windows which I would like to fix, but I will not have easy access to a Windows machine to do so until the new year.

There is also a different `gnuplot-mode` on MELPA which has been reported to work better on Windows, so you might try that too.

**Comment by BrianZbr** [comment]
Great, this information is very helpful, thanks!
